### DISCUSSION TITLE
More placeholders for custom clock format

### DISCUSSION THREAD

**Discussion by ygoe** [ramensoftware/windhawk-mods#266] [General]
I've found the clock format tweak and think it could be useful to replace T-Clock if that stops working in Windows 11 (as somebody told me recently). I'm missing some more placeholders there. Would it be possible to allow showing just the day and month of the date separately? I really don't need to see the current year every day but would like to make some room for the week number and weekday (which should already exist). My current T-Clock format for the date part is like: W %week% %weekday% %day%.%month%.

The day and month parts should also be available without leading zeros to save further space on most days of the year.

**Comment by m417z** [comment]
> W %week% %weekday% %day%.%month%

Some of it can be achieved with [the Windows date format](https://learn.microsoft.com/en-us/windows/win32/intl/day--month--year--and-era-format-pictures) - Day of the week (abbreviated or full), day and month with or without leading zeros. Weekday isn't available in the Windows format, and so the mod format can be used instead.

So you can have something like `%weekday% %date%`, while the date is configured as needed.

**Reply by ygoe** [reply]
> while the date is configured as needed

How would that look then?

**Reply by m417z** [reply]
Here's an example:

![image](https://github.com/ramensoftware/windhawk/assets/4129781/c730f02b-98bc-4627-9d0a-e34b9d17e9c2)

**Reply by ygoe** [reply]
Thank you, this works great!
